packages/@n8n/api-types/src/dto/credentials/credentials-get-many-request.dto.ts

```diff
@@ -0,0 +1,22 @@
+import { Z } from 'zod-class';
+
+import { booleanLiteral } from 'dto/common';
```

Comment:
not sure why automatic imports for this package don't use relative paths. this import will likely break sometimes.